### PR TITLE
app-text/katarakt: Katarakt is a simple PDF viewer (new package)

### PR DESCRIPTION
**Pull request by shifty91**
katarakt is a simple PDF viewer. It is designed to use as much available screen
space as possible.

There are currently two layouts. The presentation layout is very simple and
only supports scrolling on a per page basis. As the name suggests the current
page is displayed in the center and zoomed to fit the window. It is active by
default.

The grid layout is much more advanced and offers continuous (smooth, per pixel)
scrolling, zooming and adjusting the column count. Pages keep their correct
relative size and are shown in a grid.

Closes: https://bugs.gentoo.org/707872
Package-Manager: Portage-2.3.84, Repoman-2.3.20
Signed-off-by: Kurt Kanzenbach <kurt@kmk-computers.de>

### CODE REVIEW

File: app-text/katarakt/katarakt-0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils
+
+DESCRIPTION="Katarakt is a simple PDF viewer"
```
Comment (a17r) [inline app-text/katarakt/katarakt-0.2.ebuild]: No `${PN}` in the description please. `Simple PDF viewer` would be fine.

File: app-text/katarakt/katarakt-0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	app-text/asciidoc
+	virtual/pkgconfig"
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtdbus:5
+	app-text/poppler[qt5]"
```
Comment (a17r) [inline app-text/katarakt/katarakt-0.2.ebuild]: Please sort the dependencies alphanumerically.

File: app-text/katarakt/katarakt-0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${PN}-v${PV}"
+PATCHES=( "${FILESDIR}/${P}-install.patch" )
```
Comment (a17r) [inline app-text/katarakt/katarakt-0.2.ebuild]: empty line between `S` and `PATCHES` please.

**Comment by shifty91** [comment]
@a17r Sure, I contacted him about the release tarballs.

**Comment by shifty91** [comment]
@a17r There is currently no webspace available for the tarballs. Philipp will update the readme. Tags have to be sufficient atm.

**Comment by Flowdalic** [comment]
@shifty91 I saw that you put katarakt in ::guru. Would you be interested in moving it in ::gentoo? I am happy to proxy maintain it. If so, then please update this PR.

**Pull request closed without merging**